apps/browser-extension/src/content.ts

```diff
@@ -0,0 +1,4 @@
+;(() => {
```

Comment:
Is it better to keep for debugging purposses? Assuming we are not going to ship this to store

Reply:
Yeah exactly, it's part of the browser extension scripts that needs to be loaded